### DISCUSSION TITLE
Traceback (most recent call last):   File "main.py", line 4, in <module> ImportError: no module named 'os'

### DISCUSSION THREAD

**Discussion by FlamrXD** [pybricks/support#1934] [Q&A]
When I run pybricksdev run ble --name "Pybricks Hub" main.py it turns up with 
Traceback (most recent call last):   File "main.py", line 4, in <module> ImportError: no module named 'os'

Im trying to make a volume changer with pybricks but this happens

My code: 
```python
from pybricks.pupdevices import Motor
from pybricks.parameters import Port
from pybricks.tools import wait
import os
example_motor = Motor(Port.E)
for i in range(100):
    angle = example_motor.angle()/10
    print(angle)
    os.system('pactl -- set-sink-volume 0 '+angle)
    wait(200)
```

(I have little experince with python and this code is still in the works)

**Comment by BertLindeman** [comment]
Hello,

The module `os` or `uos` is not implemented on pybricks. 

Due to memory constraints, but also pybricks 3 does not run upon an operating system.
Would be hard to get pulseaudio running in this environment.

( Some assumption from my side in this. If I am wrong the pybricks team will set me straight 😃  )

Bert

**Comment by laurensvalk** [comment]
> Im trying to make a volume changer with pybricks.

You can change the volume using [`hub.speaker.volume(percentage)`](https://docs.pybricks.com/en/latest/hubs/primehub.html#pybricks.hubs.PrimeHub.speaker.volume).

> `os.system('pactl -- set-sink-volume 0 '+angle)`

This sounds like something AI would say :smile:
